.github/CONTRIBUTING.md

```diff
@@ -43,18 +43,36 @@ You can (and should) run our test suite using [*tox*].
 However, you’ll probably want a more traditional environment as well.
 We highly recommend to develop using the latest Python release because we try to take advantage of modern features whenever possible.
 
-Clone the *attrs* repository to your computer:
+First [fork](https://github.com/python-attrs/attrs/fork) the repository on GitHub.
+
+Clone the fork to your computer:
 
 ```console
-$ git clone git@github.com:python-attrs/attrs.git
+$ git clone git@github.com:<your-username>/attrs.git
 ```
 
 Or if you prefer to use Git via HTTPS:
 
 ```console
-$ git clone https://github.com/python-attrs/attrs.git
+$ git clone https://github.com/<your-username>/attrs.git
+```
+
+Then add the *attrs* repository as *upstream* remote:
+
+```console
+$ git remote add upstream git@github.com:python-attrs/attrs.git
```

Comment:
Since this remote would be read-only to most of the contributors, it's probably better to go for HTTPS in case they don't have SSH set up. If they do have SSH, though, this bit will change nothing and will still work. But I think, this recommendation is more universal.
```suggestion
$ git remote add upstream https://github.com/python-attrs/attrs.git
```

Reply:
I've checked out some manpages and it looks like we can also do this:
```suggestion
$ git remote add -t main -m main --tags upstream https://github.com/python-attrs/attrs.git
```

This one-time command will set up `.git/config` to only track the `main` branch and the tags from the `upstream` remote. This means that `git fetch upstream` without args will fetch exactly what we need w/o us having to bother specifying what exactly to fetch.

Reply:
@hynek I strongly recommend accepting the suggestion in the last comment since it'd make sure that unwanted non-`main` branches won't be fetched by the contributors, creating confusion.